uui-core/hooks/useTableState/helpers/parseUrlFiltersConfig.ts

```diff
@@ -0,0 +1,7 @@
+export const parseUrlFiltersConfig = (): Record<string, any> | undefined => {
```

Comment:
do we need separate parser for FilterConfig, may we can make generic function for all table state which is parsed to url?